src/charm.py

```diff
@@ -159,23 +157,23 @@ def _write_config_file(self, database_url: str, nrf_url: str) -> None:
         """
         jinja2_env = Environment(loader=FileSystemLoader("src/templates"))
         template = jinja2_env.get_template("smfcfg.yaml.j2")
-        # TODO: Fix config file params to write.
         content = template.render(
-            nrf_url=nrf_url,
-            smf_url=self._smf_hostname,
-            pod_id=self._pod_id,  # type: ignore[attr-defined] # noqa: E501 TODO: remove.
-            default_database_url=DATABASE_NAME,
-            smf_database=SMF_DATABASE_NAME,
+            database_name=DATABASE_NAME,
             database_url=database_url,
+            smf_url=self._smf_hostname,
+            smf_sbi_port=SMF_SBI_PORT,
+            nrf_url=nrf_url,
+            pod_ip=self._pod_ip,
         )
-        self._container.push(path=f"{BASE_CONFIG_PATH}/{CONFIG_FILE_NAME}", source=content)
-        logger.info(f"Pushed: {CONFIG_FILE_NAME} to workload.")
+        self._container.push(path=f"{BASE_CONFIG_PATH}/{CONFIG_FILE}", source=content)
+        logger.info(f"Pushed: {CONFIG_FILE} to workload.")
```

Comment:
```suggestion
        logger.info("Pushed: %s to workload.", CONFIG_FILE)
```
It is bad practice to use f-strings in logging calls.

Reply:
Fixed.